JavaIntegration/src/main/java/org/openzen/zencode/java/module/JavaNativeModule.java

```diff
@@ -205,6 +203,17 @@ public Optional<TypeSymbol> findClass(Class<?> cls) {
 		return nativeModuleSpace.getModule(cls).flatMap(module -> module.findLocalClass(cls));
 	}
 
+	public boolean isKnownType(Type type) {
+		if(type instanceof Class<?>) {
+			return isKnownType((Class<?>)type);
+		}
+		if (type instanceof ParameterizedType) {
+			return isKnownType(((ParameterizedType)type).getRawType());
+		}
+
```

Comment:
Question: Do we need the other types here as well?
We could migrate this to ZC's JavaAnnotatedType btw...